pyccel/commands/pyccel_config.py

```diff
@@ -0,0 +1,45 @@
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/devel/LICENSE for full license details.      #
+#------------------------------------------------------------------------------------------#
+""" Module containing scripts to manage compilation information.
+"""
+
+from .argparse_helpers import check_file_type, add_compiler_selection
+
+PYCCEL_CONFIG_DESCR = 'Compilation configuration management.'
+
+def setup_pyccel_config_parser(parser):
+    """
+    Add the pyccel config arguments to the parser.
+
+    Add the pyccel config arguments to the parser for command line arguments.
+
+    Parameters
+    ----------
+    parser : argparse.ArgumentParser
+        The parser to be modified.
+    """
+    subparsers = parser.add_subparsers(required=True)
+    export_parser = subparsers.add_parser('export')
+    export_parser.add_argument('filename', metavar='FILE', type=check_file_type(('.json',)),
+                        help='The file that the parser information should be exported to.')
```

Comment:
How does the command look like?
Assuming `check` is left for later:
```
$ pyccel config export FILE
$ pyccel config check FILE
```
I feel that 2 levels of subcommands is a bit too much, given that they share the same compiler flags. Why not simply using the two flags `--export` and `--check` instead?

Reply:
Nowhere in a JSON configuration file we specify the compiler family. Hence, it would make sense to check that this file is compatible with a specific compiler family, if specified

Reply:
All right, the `pyccel config` subcommand could be much more complex than I expected. I agree that having sub-sub-commands is probably the cleanest solution